### PR TITLE
ecs total number bugfix

### PR DESCRIPTION
**Pull request by WeiXuSeu**
#### What type of this PR

Add one of the following kinds:
/kind bugfix

#### What this PR does / why we need it:
cloud resource get wrong ecs total number

#### Which issue(s) this PR fixes:
bugfix cloud resource ecs total number

#### Specified Reivewers:
/assgin @luobily

### CODE REVIEW

File: modules/ops/impl/aliyun-resources/ecs/ecs.go [hunk 1]
```diff
@@ -34,18 +34,26 @@ func List(ctx aliyun_resources.Context, page aliyun_resources.PageOption,
...
 		ctx.Region = region
-		ecsList, err := DescribeResource(ctx,
-			aliyun_resources.PageOption{
-				PageSize:   &pagesize,
-				PageNumber: &pagenum,
-			}, cluster, "", IPs)
-		if err != nil {
-			return nil, 0, err
+		pagenum := 1
+		MAX := 1000
```
Comment (luobily) [inline modules/ops/impl/aliyun-resources/ecs/ecs.go:40]: MAX should not be capitalized.

File: modules/ops/impl/aliyun-resources/ecs/ecs.go [hunk 2]
```diff
@@ -55,13 +63,36 @@ func List(ctx aliyun_resources.Context, page aliyun_resources.PageOption,
...
+		logrus.Errorf("get total ecs number failed, error: %v", err)
+		return nil, 0, err
+	}
 
 	instances = instances[start:end]
 
 	return instances, total, nil
 }
 
+func Total(ctx aliyun_resources.Context, regions []string, cluster string) (int, error) {
```
Comment (luobily) [inline modules/ops/impl/aliyun-resources/ecs/ecs.go:77]: Total() is too general

Comment (luobily) [reply modules/ops/impl/aliyun-resources/ecs/ecs.go:77]: capitalized?

**Review by luobily — CHANGES_REQUESTED** [review]

File: modules/ops/impl/aliyun-resources/ecs/ecs.go (hunk 1 above)
Comment (WeiXuSeu) [reply modules/ops/impl/aliyun-resources/ecs/ecs.go]: OK，have modified

File: modules/ops/impl/aliyun-resources/ecs/ecs.go (hunk 2 above)
Comment (WeiXuSeu) [reply modules/ops/impl/aliyun-resources/ecs/ecs.go]: OK，have modified

**Review by luobily — APPROVED** [review]
lgtm

**Review by luobily — APPROVED** [review]
lgtm

**Pull request merged**